### PR TITLE
Create survivor

### PR DESCRIPTION
**Pull request by jonatasrancan**

### CODE REVIEW

File: db/models/survivor.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true
+      }
+    },
+    gender: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true,
+        isIn: { args: [['female', 'male']], msg: 'Muste female or male' }
```
Comment (talyssonoc) [inline db/models/survivor.js]: Maybe it was supposed to be `Must be female or male`?

File: routes/survivors.js [hunk 2]
```diff
@@ -11,4 +13,15 @@ router.get('/', function(req, res, next) {
     });
 });
 
+/* POST create survivor. */
+router.post('/', parseUrlencoded, function(req, res) {
+  var survivorParams = req.body;
+  console.log(req.body);
```
Comment (talyssonoc) [inline routes/survivors.js]: I think it's not supposed to be here 😛

File: routes/survivors.js [hunk 3]
```diff
@@ -11,4 +13,15 @@ router.get('/', function(req, res, next) {
...
+  db.Survivor
+    .create(req.body)
```
Comment (talyssonoc) [inline routes/survivors.js]: You could implement some form of strong parameters here.

File: routes/survivors.js [hunk 4]
```diff
@@ -1,6 +1,8 @@
 var express = require('express');
 var router = express.Router();
-var db = require('../models/index');
+var db = require('./../db/models/index');
+var bodyParser = require('body-parser');
+var parseUrlencoded = bodyParser.json({ extended: false });
```
Comment (talyssonoc) [inline routes/survivors.js]: You can use it on the base express router, like `app.use(parseUrlEncoded)` and avoid doing it all the time.

File: routes/survivors.js (hunk 2 above)
Comment (jonatasrancan) [inline routes/survivors.js]: :bomb:

File: routes/survivors.js (hunk 3 above)
Comment (jonatasrancan) [inline routes/survivors.js]: Good ideia.

**Pull request merged**